test/test-server/schema_registry_handlers.go

```diff
@@ -42,6 +42,10 @@ func handleSRUpdateTopLevelConfig(t *testing.T) http.HandlerFunc {
 			}
 			err := json.NewEncoder(w).Encode(res)
 			require.NoError(t, err)
+		case http.MethodDelete:
+			res := `{"compatibilityLevel":"BACKWARD"}`
```

Comment:
Can we serialize a struct here instead?

Reply:
this matches the behavior of the backend, returning a string. It'll be unmarshaled into a struct in the command code

Reply:
Oh right, I forgot that this API was broken. Write a comment?